spec/factories/project.rb

```diff
@@ -17,6 +18,7 @@
       project.project_time_entry_activities = FactoryBot.create_list(:time_entry_activity, 1) if project.project_time_entry_activities.empty?
 
       FactoryBot.create_list :issue, evaluator.number_of_issues, project: project
+      FactoryGirl.create_list :member, evaluator.number_of_members, :project => project, :roles => [FactoryGirl.create(:role)]
```

Comment:
Do not combine hash styles. Use prefer 1.9
https://rubocop.readthedocs.io/en/latest/cops_style/#stylehashsyntax